tests/cpp/operator/test_cast_nvfp4_transpose.cu

```diff
@@ -661,14 +661,9 @@ std::vector<std::vector<size_t>> tensor_dims = {
     {4096, 13312},
 };
 
-// Only GeLU activation tests are supported
+// Only the Identity activation is currently supported.
```

Comment:
```suggestion
// Only the Identity activation is currently tested because of a mismatch
// between `relu`/`srelu` outputs with reference implementations for NVFP4.
```